Content.Shared/_RMC14/Pulling/RMCPullingSystem.cs

```diff
@@ -237,6 +240,20 @@ private void OnPreventPulledWhileAliveAttempt(Entity<PreventPulledWhileAliveComp
         }
     }
 
+    private void OnMeleePullAttempt(Entity<MeleeWeaponComponent> ent, ref PullAttemptEvent args)
+    {
+        if (args.PullerUid != ent.Owner)
+            return;
+
+        if (ent.Comp.NextAttack > _timing.CurTime)
```

Comment:
If you can record that activating crippling strike as lurker allows you to immediately pull after a melee attack, that's the combo from 13 that Rainbeon coded